evaluation_plans/osps/vuln_management/steps.go

```
@@ -25,3 +25,19 @@ func hasSecContact(payloadData interface{}, _ map[string]*layer4.Change) (result
 
 	return layer4.Failed, "Security contacts were not specified in Security Insights data"
 }
+
+
+func sastToolDefined(payloadData interface{}, _ map[string]*layer4.Change) (result layer4.Result, message string) {
+	data, message := reusable_steps.VerifyPayload(payloadData)
+	if message != "" {
+		return layer4.Unknown, message
+	}
+
+	for _,tool := range data.Insights.Repository.Security.Tools {
+		if tool.Type == "SAST" {
```

Shame that the enum from https://github.com/ossf/security-insights-spec/blob/main/schema.cue#L116 isn't actually defined in https://github.com/ossf/si-tooling/blob/main/v2/si/security-insights.go

That isn't a critique of your choice here, more an observation of how we can improve

---

If for some reason all of the values in https://github.com/ossf/security-insights-spec/blob/main/schema.cue#L122 are `false` for the SAST tool, should this still return `layer4.Passed`?

---

oh... that's an interesting edge case, one we wouldn't expect... I'd agree we should fail on such an unexpected state

---

I added a check and updated the tests to make sure SAST is explicitly enabled. @trumant @eddie-knight